workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -50,6 +51,12 @@ private static void LaunchStandaloneClient()
             EditorApplication.delayCall += LaunchClient;
         }
 
+        [MenuItem("SpatialOS/Open inspector", false, MenuPriorities.OpenInspector)]
+        private static void OpenInspector()
+        {
+            Application.OpenURL(InspectorUrl);
```

Comment:
would be nice if we could check whether spatialos is currently running and print a warning otherwise :P

Reply:
That is a rabbit hole we are not going down (yet)

Reply:
😢